core/domain/state_domain.py

```diff
@@ -1472,7 +1472,7 @@ def validate(self, exp_param_specs_dict, allow_null_interaction):
                 'received %s' % self.solicit_answer_details)
         if self.solicit_answer_details:
             if self.interaction.id in (
-                    feconf.INTERACTION_IDS_WITHOUT_ANSWER_DETAILS):
```

Comment:
Can we remove this from feconf.py if it's not required?

Reply:
Done